### PR TITLE
tests(smoke): re-enable dialog prompt

### LINKED ISSUE #2423: Lighthouse fails to dismiss page dialogs
Lighthouse is having issues with pages that have dialogs (at least some types of dialogs) and times out when loading the page.

FWIW, WebPageTest had similar issues and [now](https://github.com/WPO-Foundation/wptagent/blob/master/internal/devtools.py#L641) tries first to "accept": false the dialog and if that fails tries "accept": true.  Similar logic in Lighthouse should make it more robust for pages with dialogs.

Test page: http://www.dlsite.com/

Log:
```
  status Loading page & waiting for onload +7ms URL, Viewport, ViewportDimensions, ThemeColor, Manifest, ChromeConsoleMessages, ImageUsage, Accessibility, EventListeners, AnchorsWithNoRelNoopener, AppCacheManifest, DOMStats, OptimizedImages, ResponseCompression, TagsBlockingFirstPaint, WebSQL
  Driver:warn confirm dialog opened by the page automatically suppressed. +4s
  method <= browser ERR:error Page.handleJavaScriptDialog +1ms 
```

### PR DESCRIPTION
**Pull request by Hoten**
This was disabled here: #2437

image

Let's see what happens in Travis these days.

fixes #2423

### CODE REVIEW

**Comment by Hoten** [comment]
If this works out, should close #2423.

**Comment by Hoten** [comment]
dbw passed fine but some totally unrelated smokes failed ...

File: lighthouse-cli/test/fixtures/dobetterweb/dbw_tester.html [hunk 1]
```diff
@@ -198,8 +198,7 @@ <h2>Do better web tester page</h2>
   https://github.com/GoogleChrome/lighthouse/issues/1194 -->
 <script>window.Error = function(error) { this.stack = 'stacktrace'; };</script>
 
-<!--The javascriptDialogOpening test is disabled as we hit a platform bug frequently on travis-->
-<!--<script>window.confirm('Is DBW Mega Tester the best site?')</script>-->
+<script>window.confirm('Is DBW Mega Tester the best site?')</script>
```
Comment (brendankenny) [inline lighthouse-cli/test/fixtures/dobetterweb/dbw_tester.html]: can you add a comment as to the purpose of this line

**Comment by brendankenny** [comment]
also does that mean #2423 is fixed?

File: lighthouse-cli/test/fixtures/dobetterweb/dbw_tester.html (hunk 1 above)
Comment (paulirish) [inline lighthouse-cli/test/fixtures/dobetterweb/dbw_tester.html]:
```suggestion
<!-- Does the automatic dialog suppression `driver.dismissJavaScriptDialogs()` work? -->
<script>window.confirm('Is DBW Mega Tester the best site?')</script>
```

**Pull request merged**